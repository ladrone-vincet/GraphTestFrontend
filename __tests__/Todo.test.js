import Todo from "../js/components/todo.js"
import React from 'react'
import renderer from 'react-test-renderer'
import Store from '../js/store.js'
jest.mock('../js/store.js')

describe("Todo", () => {
  it("should render test todo", ()=> {
    // let todo = {
    //   title: "Test title",
    //   completed: false
    // }
    // let store = {
    //   setCompleted: () => {
    //     todo.completed = true;
    //   }
    // }
    let store = new Store()
    let component = renderer.create(
      <Todo store={store} todo={store.todos[0]}/>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot()
    
  })
})
