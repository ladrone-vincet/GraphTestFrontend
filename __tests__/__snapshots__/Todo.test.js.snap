// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo should render test todo 1`] = `
<div
  className="todo"
>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
  Test title
   : 
  false
</div>
`;
